package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@iota/core": "1.0.0-beta.30",
     "@iota/http-client": "1.0.0-beta.30",
-    "@iota/mam.js": "github:iotaledger/mam.js#chrysalis",
+    "@iota/mam.js": "github:iotaledger/mam.js#main",
```

Comment:
was there any specific reason to switch back from Chrysalis into the older branch?

Reply:
They merged from the crysalis branch into main